documentation/projects/proposals/ingestion_server_removal/20240319-project_proposal.md

```diff
@@ -0,0 +1,105 @@
+# 2024-03-19 Project Proposal: Removal of the Ingestion Server
+
+**Author**: @stacimc
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @Aetherunbound
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Currently, our data refresh process is _orchestrated_ by Airflow, but the work
+is actually performed on a bespoke ingestion server. This server is relatively
```

Comment:
Perhaps we could link to it: https://github.com/WordPress/openverse/tree/fc38553d04cde7586ce680543757d77a15be9a41/ingestion_server